assets/scripts/info_bubble/RemoveButton.jsx

```diff
@@ -40,6 +41,10 @@ class RemoveButton extends React.PureComponent {
         title={this.props.intl.formatMessage({ id: 'tooltip.remove-segment', defaultMessage: 'Remove segment' })}
         onClick={this.onClick}
       >
+        <FontAwesomeIcon
+          icon="trash-alt"
+          className="remove-icon"
+          title={this.props.intl.formatMessage({ id: 'btn.remove', defaultMessage: 'Remove' })} />
```

Comment:
We may not actually need the title on the icon here, since the button itself has a title attribute already (line 41).